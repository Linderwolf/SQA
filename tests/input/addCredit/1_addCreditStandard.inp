fullstandard
addcredit
1
logout
