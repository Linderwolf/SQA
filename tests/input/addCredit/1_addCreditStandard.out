Welcome to Vapour! Please enter your username:
Username accepted! How may we help you today?
How much credit would you like to add to your account?
